packages/protocol/src/data/inscription.ts

```diff
@@ -497,11 +498,15 @@ export interface InscriptionValidation {
   path: string;
   severity: Severity;
 }
-export interface InscriptionAction {
-  kind: string;
-  payload: string;
+export interface InscriptionActionArgs {
+  actionId: "openEndPage" | "newProcess" | "newHtmlDialog" | "openCustomField" | "openPage";
+  payload: string | OpenCustomField;
   pid: PID;
```

Comment:
can't easily establish a strict correlation from the 'id' to the matching 'payload' ... but still it's getting very comfortable here

Reply:
I think it is ok if we do that manually, to at least help adopters of these actions. But definitely fine as a start.